core/neat/settings/_settings.scss

```diff
@@ -0,0 +1,20 @@
+/// Neat default grid.
+///
+/// @type map
+///
+/// @property {number (unitless)} columns 12
+///   Number of grid columns
+///
+/// @property {number (with unit)} gutter 20px
+///   Grid gutter width.
+///
+/// @property {number (unitless)} max-width 1200px
+///   Grid container's max-width.
+///
+/// @access private
+
+$neat-default-grid: (
+  columns: 12,
+  gutter: 20px,
+  max-width: 1200px,
```

Comment:
While I agree with your statement here, I feel like people would think of this as a core feature. It seems silly since its one line of code but I feel like this would meet peoples expectations.